09-privilege-escalation/index.md

```diff
@@ -0,0 +1,145 @@
+## Introduction
```

Comment:
```suggestion
---
linkTitle: 09. Privilege Escalation
type: docs
weight: 10
---
# Introduction```

We need to add this header in order for the Docsy framework to render the information on the web page. [1]
You can see an example here. [2]
[1]: https://security-summer-school.github.io/
[2]: https://raw.githubusercontent.com/security-summer-school/web/master/web-basics-browser-security-model/index.md

Reply:
@cataiovita 

Please add a header like this in the `index.md`. file.